tokio-quiche/src/quic/connection/mod.rs

```diff
@@ -140,6 +143,7 @@ pub struct Incoming {
     /// If set, then `buf` is a GRO buffer containing multiple packets.
     /// Each individual packet has a size of `gso` (except for the last one).
     pub gro: Option<u16>,
+    pub path_id: u64,
```

Comment:
So I dig in further in the code, and there are two points.

First, this is not the Path ID at the QUIC level, but instead the socket ID on which this incoming packet was received. My previous comments about handling Path ID gaps do not hold anymore, as the different `Vec` that are found regarding the different (network) paths are with regard to the sockets. It may be reasonable then to keep the `Vec` as they are currently -- as we can assume the current tokio-quiche client will not support network mobility.

Second, if I comment this line, I see that unless when exchanging the data with the `PollRecvData`, this field is used nowhere. Therefore, what is the point of tracking this?